detection_rules/etc/non-ecs-schema.json

```diff
@@ -201,20 +204,23 @@
   "logs-azure.graphactivitylogs-*": {
     "azure.graphactivitylogs.properties.c_idtyp": "keyword",
     "azure.graphactivitylogs.properties.user_principal_object_id": "keyword",
-    "azure.graphactivitylogs.properties.requestUri": "keyword"
+    "azure.graphactivitylogs.properties.requestUri": "keyword",
+    "azure.graphactivitylogs.properties.c_sid": "keyword"
   },
   "logs-azure.auditlogs-*": {
     "azure.auditlogs.properties.target_resources.0.modified_properties.1.display_name": "keyword",
     "azure.auditlogs.properties.target_resources.0.modified_properties.1.new_value": "keyword",
     "azure.auditlogs.properties.target_resources.0.modified_properties.3.new_value": "keyword",
     "azure.auditlogs.properties.target_resources.0.modified_properties.2.new_value": "keyword",
     "azure.auditlogs.properties.additional_details.value": "keyword",
-    "azure.auditlogs.properties.target_resources.0.modified_properties.0.new_value": "keyword"
+    "azure.auditlogs.properties.target_resources.0.modified_properties.0.new_value": "keyword",
+    "azure.auditlogs.properties.target_resources.0.modified_properties.0.old_value": "keyword"
   },
   "logs-azure.platformlogs-*": {
     "azure.platformlogs.identity.claim.upn": "keyword",
     "azure.platformlogs.properties.id": "keyword",
-    "azure.platformlogs.identity.claim.appid": "keyword"
+    "azure.platformlogs.identity.claim.appid": "keyword",
+    "azure.platformlogs.identity.claim.objectid": "keyword"
```

Comment:
I dont see `azure.platformlogs.identity.claim.objectid` in our telem looking back 1 year. Ill need to verify this manually via emulation and checking the logs.

Reply:
Fixed this by removing `azure.platformlogs.identity.claim.objectid` and updating rule `c07f7898-5dc3-11f0-9f27-f661ea17fbcd` by removing reference to that field. Double checked query.

<img width="1697" height="1209" alt="Screenshot 2025-09-26 at 3 03 20 PM" src="https://github.com/user-attachments/assets/8c040494-4f74-4cb4-96c3-ddac131650d9" />